package.json

```diff
@@ -54,17 +54,19 @@
   },
   "homepage": "https://github.com/solidos/solid-ui",
   "dependencies": {
+    "@inrupt/universal-fetch": "^1.0.1",
```

Comment:
will leave it in dependencies instead of dev because I believe we will need it in other places so that we can use node 18.

Reply:
Thought I would leave it until we make a decision.

Note: this is a node 16 problem with the clearImmediate function affecting the autocompleteField tests.